galactic/distribution.yaml

```diff
@@ -2631,6 +2631,16 @@ repositories:
       url: https://github.com/ros-perception/perception_pcl.git
       version: foxy-devel
     status: maintained
+  performance_test-release:
+    release:
```

Comment:
could you add a source and a doc entry?

Reply:
@ivanpauno I have pushed the changes. Thank you!